ruma-appservice-api/src/event/push_events/v1.rs

```diff
@@ -36,6 +44,84 @@ impl<'a> Request<'a> {
     }
 }
 
+#[derive(Debug, Deserialize)]
+struct EventDeHelper {
```

Comment:
You can declare this inside the conversion function (so you only need one `#[cfg]` attribute to make this feature optional).